src/fides/api/task/graph_task.py

```diff
@@ -154,61 +154,75 @@ def result(*args: Any, **kwargs: Any) -> Any:
                     logger.warning(
                         "Retrying {} {} in {} seconds...",
                         method_name,
-                        self.traversal_node.address,
+                        self.execution_node.address,
                         func_delay,
                     )
                     sleep(func_delay)
                     raised_ex = ex
             self.log_end(action_type, raised_ex)
-            self.resources.request.cache_failed_checkpoint_details(
-                step=action_type, collection=self.traversal_node.address
-            )
+            # This is useful for resuming from the access, erasure, or consent
+            # checkpoints if the graph fails in one of these sections.
+            self.resources.request.cache_failed_checkpoint_details(step=action_type)
             add_errored_system_status_for_consent_reporting(
                 self.resources.session,
                 self.resources.request,
                 self.connector.configuration,
             )
-            # Re-raise to stop privacy request execution on failure.
-            raise raised_ex  # type: ignore
+            if not self.request_task.id:
```

Comment:
correct, I'm checking the request task instead of the config variable because I have some extra logic that switches the scheduler if the privacy request was in progress before DSR 3.0 was added -


for example, say we are running a Privacy Request on DSR 2.0 and it fails.  Then customer upgrades and they start using DSR 3.0.  If this privacy request is reprocessed, it will still run on DSR 2.0 -